{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/theme/webpack.py

```diff
@@ -16,6 +16,9 @@
             dependencies={
                 # add any additional npm dependencies here...
             }
-        )
+        ),
+        aliases={
```

Comment:
since it is inside a dict, shouldn't it be a string?
```suggestion
        'aliases': {
```

Reply:
Thanks, indeed I missed a bracket there :)